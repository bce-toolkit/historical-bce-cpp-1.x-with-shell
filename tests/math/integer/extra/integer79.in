55903222024971792486695964965003
4221236152272760193071812934725109496497839257490024996529941679100866645276547603627466807
