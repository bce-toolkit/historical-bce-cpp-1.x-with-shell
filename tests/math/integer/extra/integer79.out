4221236152272760193071812934725109496497839257490024996529997582322891617069034299592431810
-4221236152272760193071812934725109496497839257490024996529885775878841673484060907662501804
235980701840341750857950717715225822585474464144133739406143246412031246911810893064896266760924698422472265654183299155421
0
55903222024971792486695964965003
0
1
1
1
0
0
